roles/oem/tasks/vm_only_post.yml

```
@@ -53,11 +53,11 @@
 # Extensions which are non-free software and available only under the Oracle
 # PUEL. We should ensure we have not accidentally included them.
 - name: Validate missing USB 2/3 controllers  # noqa risky-shell-pipe
-  command: lspci
+  ansible.builtin.command: lspci
```

Looks like a few other uses have been refactored to use the longer form. Should we be consistent about that? I'll mark the others as suggestions in case that's the direction we want to go.

```suggestion
  ansible.builtin.command:
    cmd: lspci
```